holidays/at.yaml

```diff
@@ -1,25 +1,152 @@
 ---
 
+_nominatim_url: 'https://nominatim.openstreetmap.org/reverse?format=json&lat=48.2083537&lon=16.3725042&zoom=18&addressdetails=1&accept-language=de,en'
+
 PH:  # https://de.wikipedia.org/wiki/Feiertage_in_%C3%96sterreich
   'Neujahrstag': [1, 1]
   'Heilige Drei Könige': [1, 6]
-  # 'Josef': [3, 19, [Kärnten, Steiermark, Tirol, Vorarlberg]]
-  # 'Karfreitag': [easter, -2]
   'Ostermontag': [easter, 1]
   'Staatsfeiertag': [5, 1]
-  # Florian: [5, 4, [Oberösterreich]]
   'Christi Himmelfahrt': [easter, 39]
   'Pfingstmontag': [easter, 50]
   'Fronleichnam': [easter, 60]
   'Mariä Himmelfahrt': [8, 15]
-  # 'Rupert': [9, 24, [Salzburg]]
-  # 'Tag der Volksabstimmung': [10, 10, [Kärnten]]
   'Nationalfeiertag': [10, 26]
   'Allerheiligen': [11, 1]
-  # 'Martin': [11, 11, [Burgenland]]
-  # 'Leopold': [11, 15, [Niederösterreich, Wien]]
   'Mariä Empfängnis': [12, 8]
-  # 'Heiliger Abend': [12, 24]
   'Christtag': [12, 25]
   'Stefanitag': [12, 26]
-  # 'Silvester': [12, 31]
+SH:  # https://www.help.gv.at/Portal.Node/hlpd/public/content/11/Seite.17602001.html
```

Comment:
I guess also in the case of Austria there is no Open Data available. Where you able to automate the process somehow?